app/javascript/react/store/fixedSessionsSelectors.ts

```diff
@@ -1,91 +1,158 @@
 import { createSelector } from "reselect";
-
 import { StatusEnum } from "../types/api";
 import { Session, SessionList } from "../types/sessionType";
 import { RootState } from "./";
+import { FixedSession } from "./fixedSessionsSlice";
+import { FixedSessionsTypes } from "./sessionFiltersSlice";
+
+const selectActiveFixedSessionsState = (state: RootState) =>
+  state.fixedSessions.activeSessions;
+
+const selectDormantFixedSessionsState = (state: RootState) =>
+  state.fixedSessions.dormantSessions;
 
-const selectFixedSessionsState = (state: RootState) => state.fixedSessions;
 const selectFixedSessionsStatusFulfilled = (state: RootState) =>
   state.fixedSessions.status === StatusEnum.Fulfilled;
 
-const selectFixedSessionsPoints = createSelector(
+const selectFixedSessionsState = (state: RootState) => state.fixedSessions;
+
+const selectIsActiveSessionsFetched = createSelector(
+  [selectFixedSessionsState],
+  (fixedSessionsState) => fixedSessionsState.isActiveSessionsFetched
+);
+
+const selectIsDormantSessionsFetched = createSelector(
   [selectFixedSessionsState],
-  (fixedSessionsState): Session[] =>
-    fixedSessionsState.sessions.map(
+  (fixedSessionsState) => fixedSessionsState.isDormantSessionsFetched
+);
+
+const transformSessionData = (sessions: FixedSession[]) =>
+  sessions.map(
+    ({
+      id,
+      title,
+      lastMeasurementValue,
+      startTimeLocal,
+      endTimeLocal,
+      latitude,
+      longitude,
+      streams,
+    }) => {
+      const firstStream = streams[Object.keys(streams)[0]];
```

Comment:
what will happen if `streams` is empty or the key lookup fails? Adding a fallback could be a good idea.